packages/patterns/src/components/category-editor.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useMemo, useState } from '@wordpress/element';
+import { FormTokenField } from '@wordpress/components';
+import { useDebounce } from '@wordpress/compose';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import CategorySelector from './category-selector';
+
+const unescapeString = ( arg ) => {
+	return decodeEntities( arg );
+};
```

Comment:
```suggestion
const unescapeString = ( arg ) => decodeEntities( arg );
```

Totally optional. Just a style thing

Reply:
ah, yeh, thanks, I think that was a copy pasta that wasn't tidied up, done now.